.circleci/config.yml

```diff
@@ -28,14 +30,33 @@ jobs:
       - restore_cache:
           keys:
             - v1-pre-commit-dot-cache-{{ checksum ".pre-commit-config.yaml" }}
+      - restore_cache:  # ensure this step occurs *before* installing dependencies
+          key: v1-pipenv-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
```

Comment:
What does adding the branch key here do?

Reply:
You know, I found it online and just copied it in, but yeah, it probably doesn't need to be there, does it?

I think I'll remove it. Thank you